Gds/src/fprime_gds/common/pipeline/standard.py

```diff
@@ -103,7 +103,8 @@ def setup_logging(self, log_dir):
         """
         Setup logging based on the logging prefix supplied
 
-        :param prefix: logging prefix to use
+        :param log_dir: TODO: the parameter description should be filled by maintainer
```

Comment:
prefix was most likely replaced by log_dir. We can use the following description here:
`logging output directory`